packages/lesswrong/components/users/SignupSubscribeToCurated.tsx

```diff
@@ -34,6 +35,10 @@ const SignupSubscribeToCurated = ({ defaultValue, onChange, classes }: {
   classes: ClassesType,
 }) => {
   const [checked, setChecked] = useState(defaultValue);
+
+  const emailType = forumTypeSetting.get() === 'EAForum' ?
+    'the EA Forum weekly digest email' : 'Curated posts';
+
   return <div className={classes.root}>
```

Comment:
Jesus, I'm kinda annoyed they reused this outside of the signup context. I guess it's fine? Add a comment here why we're modifying this. (We don't use it in our signup flow)